### ISSUE TITLE
Bug on createMutation via OGM

### ISSUE THREAD

**Issue by brechtarnou** [neo4j/graphql#288]
**Describe the bug**
When you try to create a type for example a USER and a USER has a relationship with another label COMPANY, but on the otherhand also has a propery starting with COMPANY for example COMPANYID makes it impossible to add COMPANYID to the user. If you try to add the COMPANYID the result is always null. However I tried by reversing COMPANYID to IDCOMPANY and than the ID is filled in correctly.

**Type definitions**
```graphql
  type USER @exclude(operations: [CREATE, UPDATE, DELETE]) {
    USERID: String
    EMAIL: String
    COMPANYID: String
    ROLE: Role
    CREATEDON: String
    COMPANY: [COMPANY] @relationship(type: "IS_PART_OF", direction: OUT)
  }
```

**To Reproduce**
Create a type for example like above USER
Add a relationship with COMPANY
Add a property starting with COMPANY (e.g. COMPANYID)
Try to create a user and add a COMPANYID

**Expected behavior**
Company should be filled in.

Thanks in advance!

**Comment by darrellwarde** [comment]
This is going to be fixed in version 2.0.0, and the fix for version 1.x isn't so great, so we'll hold off until the 2.0.0 release for this one.

**Comment by darrellwarde** [comment]
In an effort to clean our issue backlog a bit, going to close this one off as the fix is available in any 2.0.0 release whenever you decide to upgrade, be it during prerelease or after stable.
